deployment/clusterObjectModel/cluster_object_model.py

```diff
@@ -57,15 +56,13 @@ def get_service_model_list(self):
                 sub_model_list.append(sub_dir_name)
         return sub_model_list
 
-
-
-    def init_service_parser(self, service_name):
+    def get_service_parser(self, service_name):
 
         sys.path.insert(0, '{0}/../../src/{1}/config'.format(package_directory_com, service_name))
         default_path = "{0}/../../src/{1}/config/{1}.yaml".format(package_directory_com, service_name)
 
         # Prepare Service Configuration
-        cluster_cfg = self.cluster_configuration
+        layout = self.layout
```

Comment:
>layout [](start = 8, length = 6)

At least, I'm very confused by the meaning of layout.:grinning:

Reply:
As my previously design, it's only a map of host -> labels, and services are aligned to labels, so we decouple services from nodes.

There is some feedback that want to keep the nodes info in one place. So finally the layout include other info, for example: k8s-role, sku.

Well, it comes out nearly the same as the cluster-configuration.yaml.

Reply:
we can now remove k8s-role. Because actually it's useless. I will do it in the future.